src/lib/components/comparisonTable/components/TableButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from './style.module.scss';
+
+interface Props {
+  onClick: () => void;
+  className?: string;
+}
+
+const TableButton: React.FC<Props> = ({
+  children,
+  onClick,
+  className = '',
+}) => (
+  <span
+    className={`${styles.button} ${className}`}
+    data-testid="ds-table-button"
+    onClick={onClick}
+    role="button"
+    tabIndex={0}
+  >
```

Comment:
Any reason to use `span` instead of a `button` tag?
I'm not a big fan of adding button capabilities to non interactive elements, if possible I'd suggest switching this to a regular `button` and fix styles if necessary.
What do you think?

Reply:
Not really, I was following the other table button example but if thinking critically about it there's no reason for this to be this way. Just updated, also added an outline color based on the button outline for accessibility purposes.